DEVELOPMENT.md

```diff
@@ -22,7 +22,7 @@ Please download all the `fhir-works-on-aws` package and place them inside one di
 
 Copy these files to your parent directory
 
-- [fhir-works-local-develop.sh](./scripts/fhir-works-deploy.sh)
+- [fhir-works-local-develop.sh](./scripts/fhir-works-local-deploy.sh)
```

Comment:
Looks like there's no such file. It's either `fhir-works-local-develop.sh` or `fhir-works-deploy.sh`

Reply:
```suggestion
- [fhir-works-local-develop.sh](./scripts/fhir-works-local-develop.sh)
```